vizro-core/src/vizro/models/_components/container.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Literal
+
+from dash import html
+
+try:
+    from pydantic.v1 import Field, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, validator
+
+from vizro.models import VizroBaseModel
+from vizro.models._models_utils import (
+    _assign_component_grid_area,
+    _create_component_container,
+    _log_call,
+    set_components,
+    set_layout,
+)
+from vizro.models.types import ComponentType
+
+if TYPE_CHECKING:
+    from vizro.models import Layout
+
+
+class Container(VizroBaseModel):
+    """Html.Div that can wrap other `Page.components`.
```

Comment:
```suggestion
    """Container to group together a set of components on a page.
```

`html.Div` is quite technical/not something that most people actually care about.

It's also arguably not what we should be using - probably we should favour [semantic elements like `<section>` instead](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section). The same could be said for a lot of our divs, like `<nav>`. It's not important or worth changing anywhere now but something for us to have in the back of our minds I think.

It does make me think that `Section` is possibly a better name than `Container` though 🤔 Even though we discussed naming of this to death already, actually I'm not sure we considered that name before...

Reply:
I've implemented the docstring, but for the semantic elements point or even re-naming - let's discuss that on Monday 😄